packages/@react-stately/radio/src/useRadioGroupState.ts

```diff
@@ -93,6 +96,7 @@ export function useRadioGroupState(props: RadioGroupProps): RadioGroupState  {
     isReadOnly: props.isReadOnly || false,
     isRequired: props.isRequired || false,
     validationState: props.validationState || (isInvalid ? 'invalid' : null),
-    isInvalid
+    isInvalid,
+    excludeFromTabOrder: props.excludeFromTabOrder
```

Comment:
this shouldn't be in the state, we have a "hook context" in useRadioGroup, that should be an ok place to propagate this so all the useRadio's can read the value

https://github.com/adobe/react-spectrum/blob/b46d23b9919eaec8ab1f621b52beced82e88b6ca/packages/%40react-aria/radio/src/useRadioGroup.ts#L127

Reply:
Thank you for guiding me. Fixed the code.